src/Microsoft.Management.Configuration.Processor/Microsoft.Management.Configuration.Processor.csproj

```diff
@@ -73,29 +73,23 @@
     <CsWinRTComponent>false</CsWinRTComponent>
     <CsWinRTIncludes>
       Microsoft.Management.Configuration;
-      Windows.ApplicationModel;
-      Windows.Data;
-      Windows.Devices;
+      Windows.Data.Text.TextSegmen;
+      Windows.Devices.Geolocation;
       Windows.Foundation;
-      Windows.Globalization;
-      Windows.Graphics;
-      Windows.Management;
-      Windows.Media;
-      Windows.Networking;
-      Windows.Perception;
-      Windows.Security;
-      Windows.Services;
+      Windows.Globalization.DayOfWee;

```

Comment:
I love this. It sounds so festive to me. "Day of weeeeee~"
But brits may disagree with my interpretation 😆